src/index.html

```diff
@@ -2,15 +2,44 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <section class="search">
+      <form
+      class="search-form search__bar--lg"
+      action="#"
+      method="post"
+      data-qa="big"
+      >
```

Comment:
Fix it in the whole file.
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54860641/21d5a321-1f6f-42fc-a60d-69af3da90164)

Reply:
done